MaterialLib/SolidModels/CreateLubby2.h

```diff
@@ -11,14 +11,36 @@
 
 #include <logog/include/logog.hpp>
 
+#include "ProcessLib/Utils/ProcessUtils.h"  // required for findParameter
+
 #include "Lubby2.h"
 #include "MechanicsBase.h"
-#include "ProcessLib/Utils/ProcessUtils.h"  // required for findParameter
 
 namespace MaterialLib
 {
 namespace Solids
 {
+namespace Lubby2
+{
+inline NumLib::NewtonRaphsonSolverParameters
+createNewtonRaphsonSolverParameters(BaseLib::ConfigTree const& config)
+{
+    DBUG("Create local nonlinear solver parameters.");
+    int const maximum_iterations =
+        //! \ogs_file_param{material__solid__constitutive_relation__Lubby2__nonlinear_solver__maximum_iterations}
+        config.getConfigParameter<int>("maximum_iterations");
+
+    DBUG("\tmaximum_iterations: %d.", maximum_iterations);
+
+    double const error_tolerance =
+        //! \ogs_file_param{material__solid__constitutive_relation__Lubby2__nonlinear_solver__error_tolerance}
+        config.getConfigParameter<double>("error_tolerance");
+
+    DBUG("\terror_tolerance: %g.", error_tolerance);
+
+    return {maximum_iterations, error_tolerance};
+}
```

Comment:
I'm aware of the redundancy. This has to do with the way the documentation of the project file parameters is done. Maybe it's a good point to start thinking about documentation improvements.
Keeping this as it is for now.